### PR TITLE
Add missing tests in StringyTest

### PR DESCRIPTION
**Pull request by vlakoff**
For consistency with StaticStringyTest.

### CODE REVIEW

File: tests/StringyTest.php [hunk 1]
```diff
@@ -546,6 +546,19 @@ public function testTruncate($expected, $str, $length, $substring = '',
     }
 
     /**
+     * @dataProvider safeTruncateProvider()
+     */
+    public function testSafeTruncate($expected, $str, $length, $substring = '',
+                                     $encoding = null)
```
Comment (GrahamCampbell) [inline tests/StringyTest.php]: are we following psr-2?

Comment (GrahamCampbell) [inline tests/StringyTest.php]: because that's not psr-2

Comment (danielstjules) [inline tests/StringyTest.php]: I think 4.4 of psr-2 is one of the only things we hadn't been following.

The difference:

``` php
public function testSafeTruncate($expected, $str, $length, $substring = '',
                                 $encoding = null)
{
    // body
}

public function testSafeTruncate(
    $expected,
    $str,
    $length,
    $substring = '',
    $encoding = null
) {
    // body
}
```

Comment (vlakoff) [inline tests/StringyTest.php]: How about merging this PR and, if you wish, opening a separate issue for further discussion concerning CS?

**Pull request merged**